src/main/java/com/google/starfish/services/NoteService.java

```diff
@@ -60,7 +41,7 @@ public long getNumFavoritesById(DataSource pool, long noteId) throws SQLExceptio
           String pdfSource = rs.getString("pdf_source");
           Date dateCreated = rs.getDate("date_created");
           long numDownloads = rs.getLong("num_downloads");
-          long numFavorites = getNumFavoritesById(pool, noteId);
+          long numFavorites = favoriteNoteService.getNumFavoritesByNoteId(pool, noteId);
```

Comment:
Could you add the misc labels for the note here?

Reply:
Yes will do that.